js/jquery.ntm.js

```diff
@@ -53,7 +53,13 @@
                     }
                 }
 
-                //Thinesh - Explicitly hide the code that children nodes are expanded automatically next refresh.
+                //Thinesh - Add id for parent node
+                if (options.autoParentDetection) {
+                    parent.attr('id','parentnode-' + parentindex);

```

Comment:
Sorry Sudhir. I missed that. I will update you.

On Wed, Apr 20, 2016 at 4:12 PM, SudhirR <notifications@github.com> wrote:

> In js/jquery.ntm.js
> <https://github.com/mws-public/Nxeed-Tree-Menu/pull/5#discussion_r60386222>
> :
>
> > @@ -53,7 +53,13 @@
> >                      }
> >                  }
> >
> > -                //Thinesh - Explicitly hide the code that children nodes are expanded automatically next refresh.
> > +                //Thinesh - Add id for parent node
> > +                if (options.autoParentDetection) {
> > +                    parent.attr('id','parentnode-' + parentindex);
>
> where is parentindex 1. declared and 2. initialized?
> Also parentindex needs to be declared as parentIndex.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mws-public/Nxeed-Tree-Menu/pull/5/files/79dc4d2ea742f077a9b5370b51376a95389513ca#r60386222>
>